rr_iarrc/src/finish_line_watcher/finish_line_watcher.cpp

```diff
@@ -110,21 +194,20 @@ int main(int argc, char** argv) {
     nhp.param("blockWheels_height", blockWheels_height, 800);
     nhp.param("blockBumper_height", blockBumper_height, 800);
 
+    nhp.param("publish_when_detected", publish_when_detected, true);
+
+    nhp.param("min_contour_area", min_contour_area, 5.0);
+    nhp.param("angle_cutoff", angle_cutoff, 20.0);
+    nhp.param("width_cutoff", width_cutoff, 275.0);
+
     ROS_INFO("Finish line watching %s", img_topic.c_str());
 
-    Subscriber img_saver_sub = nh.subscribe(img_topic, 1, ImageCB);
+    ros::Subscriber img_saver_sub = nh.subscribe(img_topic, 1, ImageCB);
 
     crosses_pub = nh.advertise<std_msgs::Int8>("finish_line_crosses", 1);
+    debug_pub = nh.advertise<sensor_msgs::Image>("finish_line_detection/debug", 1);
```

Comment:
Change it to this `nhp.advertise<sensor_msgs::Image>("debug", 1);` so it has the same names as the node